examples/webhook-authz/README.md

```diff
@@ -0,0 +1,107 @@
+# Example Webhook authorizer
+
+This directory contains the scaffolding necessary to use a webhook authorizer
+with the standalone cluster registry. It is not meant to be used with the
+aggregated cluster registry.
+
+NOTE: this example is not meant for production use! Do not deploy this into a
+production environment.
+
+## Usage
+
+1.  `cd` into this directory.
+
+2.  Deploy a standalone cluster registry. You can use
+    [crinit](https://github.com/kubernetes/cluster-registry/blob/master/docs/userguide.md#standalone).
+
+3.  Create a self-signed SSL cert for the cluster registry webhook authorizer
+    (or skip this step and use an existing SSL cert).
+
+```sh
+openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -nodes
+```
+
+4.  Create a ConfigMap that contains the files needed for the cluster registry
+    webhook authorizer. If you deployed your cluster registry into a namespace
```

Comment:
Might be helpful to say which files these might be.

Reply:
Good point. I'll do this in a follow-up.